applications/serial_lte_modem/Kconfig

```diff
@@ -8,16 +8,10 @@ source "Kconfig.zephyr"
 
 menu "Nordic Serial LTE Modem"
 
-config SLM_CUSTOMIZED
-	bool "Flag for customized functionality"
+config SLM_CUSTOMER_VERSION
+	string "Customer version string"
 	help
-	  Enable this flag to include customized logic.
-
-config SLM_AT_MODE
-	bool "Serial LTE Modem by raw AT mode"
-	default y
-	select AT_CMD_PARSER
-	select AT_MONITOR
+	   A custom version suffix that will be appended to the SLM version string.
```

Comment:
It is not actually appended. SLM returns three different version strings. So perhaps say that it will be returned with #XSLMVER with the other version strings, similarly as you do with the documentation change.